### PR TITLE
Obs goxlr stuff

### PR DESCRIPTION
**Pull request by jdudetv**

### CODE REVIEW

File: packages/src/map.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ pkg.createNonEventSchema({
   },
   run({ ctx }) {
     const map = ctx.getInput<Map<string, any>>("map");
+    console.log(map);
```
Comment (Brendonovich) [inline packages/src/map.ts]: remove

File: packages/src/obs/requests.ts [hunk 2]
```diff
@@ -1,10 +1,53 @@
...
+  sourceHeight: s.field("Source Height", t.int()),
+  width: s.field("Width", t.int()),
+  height: s.field("Height", t.int()),
+}));
+
+interface SceneItemTransformInterface {
+  alignment: number;
+  boundsAlignment: number;
+  boundsHeight: number;
+  boundsType: string;
```
Comment (Brendonovich) [inline packages/src/obs/requests.ts:19]: I think this can be an enum, or maybe it's alignment

**Review by Brendonovich — DISMISSED** [review]

File: packages/src/map.ts (hunk 1 above)
Comment (jdudetv) [reply packages/src/map.ts]: fixed

File: packages/src/obs/requests.ts (hunk 2 above)
Comment (jdudetv) [reply packages/src/obs/requests.ts:19]: converted to enums.

**Pull request merged**